samples/batch/batch-ingestion-client/RealtimeTranscription/Helpers/RealtimeTranscriptionHelper.cs

```diff
@@ -0,0 +1,119 @@
+// <copyright file="RealtimeTranscriptionHelper.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The folder is for batch/batch-ingestion-client, not sure whether it makes sense to have RealtimeTranscription inside this folder? or should be a separate one?

Reply:
I agree, but I wanted to do that in a separate PR